torchinductor/codegen/triton.py

```diff
@@ -666,6 +666,12 @@ def indexing(
         elif not mask:
             mask = ["None"]
 
+        if mask == ["xmask"] and index == 0 and self.range_trees[0].numel == 1:
+            # This causes a triton error:
+            # https://gist.github.com/jansel/70c4b1c8041f0f27f96ee95e2edca04a
```

Comment:
@voznesenskym this is the "cannot broadcast" error that you've been encountering, so perhaps this fixes it.